tests/e2e/chain/config.go

```diff
@@ -27,6 +28,7 @@ const (
 	IbcDenom      = "ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518"
 	MinGasPrice   = "0.000"
 	IbcSendAmount = 3300000000
+	VotingPeriod  = 180000000000
```

Comment:
nit: can we add a small comment to make this more human readable? 

```suggestion
	VotingPeriod  = 180000000000 // X seconds or days or something
```